src/04-modality-specific-files/03-electroencephalography.md

```diff
@@ -13,7 +13,8 @@ and can be used for practical guidance when curating a new dataset.
 -   Multiple sessions per subject: [`eeg_rishikesh`](https://doi.org/10.5281/zenodo.1490922)
```

Comment:
The example in EEG link to "actual" data sets and not those in the bids-example repository.

Should we only point all links to the bids-example repo, given that the link to the real data is listed there anyway?

Reply:
yes, I think that'd be good! :-)